modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -1009,7 +1009,7 @@ definition {
 				panel = "Control Panel",
 				portlet = "Queue");
 
-			AssertTextPresent(value1 = "userea@liferay.com userfn userln Entry Test Edited");
+			AssertTextPresent(value1 = "userea@liferay.com userfn Test Entry Test Edited");
```

Comment:
Before the term [%CUSTOMOBJECT180339_AUTHOR_LAST_NAME%] was replaced by "userln" (This user did not create the object), but now it is being replaced correctly.